client/src/components/filterCampaigns/index.jsx

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import {
+  message,
+  Radio,
+  Input,
+  Segmented,
+
+} from 'antd';
+import './style.less';
+
+const { Search } = Input;
+const { Group } = Radio;
+function Campaigns({ setCategory, setAvailable, setSearch }) {
+  const [categories, setCategories] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const { data: { data: { categories: categoriesFromDB } } } = await axios('/api/categories');
+        const categoriesNameArr = ['List All'];
+        categoriesFromDB.map((item) => categoriesNameArr.push(item.name));
+        setCategories(categoriesNameArr);
```

Comment:
Cant you just set the category like this
```js
setCategories(['List All', ...categoriesFromDb])
```
And you can just rename the categories state categoriesList so you don't rename in line 21